README.md

```diff
@@ -2,9 +2,9 @@
 
 # Installation
 
-1. Create and configure an account at [Drift](http://www.drift.com).
-2. Add this plugin to WordPress and enable it. See [Managing Plugins](https://codex.wordpress.org/Managing_Plugins)
-3. Visit https://app.drift.com/customize and copy the code chunk
+1. Create and configure an account at [Drift](https://www.drift.com/).
+2. Add this plugin to WordPress and enable it. See [Managing Plugins](https://codex.wordpress.org/Managing_Plugins).
+3. Visit https://app.drift.com/customize and copy the code chunk.
```

Comment:
`app.drift.com/settings/widget` would be more appropriate here (it's what /customize redirects to now but regardless)